go.mod

```diff
@@ -20,6 +21,13 @@ require (
 	github.com/pion/logging v0.2.2 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	golang.org/x/crypto v0.12.0 // indirect
-	golang.org/x/sys v0.11.0 // indirect
+	golang.org/x/sys v0.12.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+exclude (
+	// note: github.com/pion/dtls/v2 must be kept at v2.2.8-0.20230828143201-609e5bee6eb0 as long as golang1.18 is supported
+	github.com/pion/dtls/v2 v2.2.8-0.20230904031456-60064c6543dd
+	// note: github.com/pion/transport/v2 must be kept at v2.2.2 as long as golang1.18 is supported
+	github.com/pion/transport/v2 v2.2.3
+)
```

Comment:
Yes, that would be much better, I wouldn't have to do the excludes then and we can use latest code. 

In fact, the current build of plgd/device and plgd/hub is broken since https://github.com/plgd-dev/go-coap/pull/473 got merged, I forgot to check the compilation of our other repos before it was merged. We were discussing with @jkralik whether to simply not update plgd/device and plgd/hub repos until we drop support for go1.18, but if you can make it compatible that will be the best case scenario. 

I can wait, no problem, it's just our convention that I update the repos at the start of a month.

Reply:
@Danielius1922 just merged! should be able to update now :)